package.json

```diff
@@ -26,18 +28,20 @@
     "react-motion": "^0.5.2",
     "react-redux": "^7.2.1",
     "react-router-dom": "5.2.0",
-    "react-scripts": "3.4.3",
+    "react-scripts": "4.0.3",
```

Comment:
I did try to go to v5. But I didn't manage it, yet. It's probably still a good idea. Or maybe switch to a different project like https://github.com/vitejs/vite.